packages/webpack-plugin/src/loader-utils.ts

```diff
@@ -17,9 +18,13 @@ export function getImports(
     const imports: string[] = [];
     const unusedImports: string[] = [];
     for (const imported of meta.getImportStatements()) {
-        if (imported.request.endsWith('.st.css')) {
+        if (imported.request.endsWith('.css')) {
```

Comment:
this was wrong in the first place. we should not check the request. we should check the resolved value.